.github/workflows/unit-tests-pr.yml

```diff
@@ -23,14 +21,14 @@ permissions:
 jobs:
   test:
     name: Unit Tests
-    if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
+    if: ${{ github.event.pull_request.state == 'opened' || github.event.review.state == 'APPROVED' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'run-tests')) }}
     uses: ./.github/workflows/frontend-unit-tests.yml
     secrets:
       codecov_token: ${{ secrets.CODECOV_TOKEN }}
 
   load-test:
     name: Load Tests
-    if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
+    if: ${{ github.event.pull_request.state == 'opened' || github.event.review.state == 'APPROVED' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'run-tests')) }}
```

Comment:
I don't know if it is possible but it would be great to put some names on these conditions and make them more clear and reusable :)

Reply:
+1

Reply:
Fixed!